front-end/src/App.css

```diff
@@ -1,6 +1,18 @@
 .App {
   text-align: center;
 }
+/* .amplify-button[data-variation='primary'] {
```

Comment:
No need to include commented out code when checking in your code.

Reply:
comments removed from App.css